spec/casher_spec.rb

```diff
@@ -23,4 +23,16 @@
       subject.run('fetch', tgz_url, tbz_url)
     end
   end
+
+  context 'when the curl call times out' do
+    before :each do
+      expect(subject).to receive(:system).with(/curl\b.*#{tgz_url.gsub('?','\?')}/m).and_raise(TimeoutError)
+    end
+
+    it 'does not clean up the fetch_tar' do
```

Comment:
Now that this works, should the description be `'cleans up the fetch_tar'`? :grin:

Reply:
AHA good catch! :D :D :D